src/NuGet.Clients/NuGet.PackageManagement.UI/IItemLoader.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace NuGet.PackageManagement.UI
 {
-    internal interface IItemLoaderState
+    public interface IItemLoaderState
```

Comment:
internals ARE visible to `NuGet.PackageManagement.UI.Test` due to assembly property declared here https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Clients/NuGet.PackageManagement.UI/Properties/AssemblyInfo.cs#L12